tests/Feature/controller/AddValidationTest.php

```diff
@@ -17,28 +19,27 @@ class AddValidationTest extends TestCase
     protected function assertFalseState($request_array)
     {
         $response = $this->call('POST', '/add', $request_array);
-        $data = json_decode($response->getContent(), true);
+        $data = json_decode($response ->getContent(), true);
         $this->assertEquals(200, $response->status());
-        if ($data['error']==false) {
+        if ($data['error']) {
+            $this->assertTrue(true);
```

Comment:
This assertion will always succeed. Please replace with something more meaningful